### PR TITLE
Adding in missing documentation in Pages capability

### PR DESCRIPTION
**Pull request by jadahiya-MSFT**

### CODE REVIEW

File: packages/teams-js/src/public/pages.ts [hunk 1]
```diff
@@ -486,12 +529,17 @@ export namespace pages {
...
+     * @returns true if the pages.backStack capability is enabled in runtime.supports.pages.backStack and
+     * false if it is disabled
      */
     export function isSupported(): boolean {
       return runtime.supports.pages ? (runtime.supports.pages.backStack ? true : false) : false;
     }
   }
 
+  /**
+   * Namespace to interact with the full-trust part of the SDK
```
Comment (erikadoyle) [inline packages/teams-js/src/public/pages.ts:541]: Could we make this description more specific (for example, brief summary of full trust)?

File: packages/teams-js/src/public/pages.ts [hunk 2]
```diff
@@ -516,6 +564,8 @@ export namespace pages {
     }
     /**
      * Checks if pages.fullTrust capability is supported currently
```
Comment (erikadoyle) [inline packages/teams-js/src/public/pages.ts:566]: Checks if the current application host supports the pages.fullTrust capability

**Review by erikadoyle — COMMENTED** [review]
Thanks @jadahiya-MSFT, overall looks good! We (Josh and Erika, from docs team) added a few suggestions. In general, would it be possible to standardize isSupported descriptions to something a little more specific (than "supported currently"), like:
Checks if the current application host supports the *x* capability

**Comment by jadahiya-MSFT** [comment]
> * in

This seems possible, I'll make these changes now :)

File: packages/teams-js/src/public/pages.ts [hunk 3]
```diff
@@ -171,7 +190,9 @@ export namespace pages {
   }
 
   /**
-   * Checks if page capability is supported currently
+   * Checks if pages capability is supported currently
```
[the remark this answers is not in the document]
Comment (erinha) [reply packages/teams-js/src/public/pages.ts:193]: I think this is a good suggestion.

File: change/@microsoft-teams-js-393fc2b1-d5c9-413a-98d5-b44bf8a88903.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Added in missing documentation to the Pages capability",
```
Comment (erinha) [inline change/@microsoft-teams-js-393fc2b1-d5c9-413a-98d5-b44bf8a88903.json:3]: nit: I don't think we need "in"

**Review by erinha — CHANGES_REQUESTED** [review]
We're getting there! Still some unaddressed feedback from Erika for improved comments.

File: packages/teams-js/src/public/pages.ts [hunk 5]
```diff
@@ -537,13 +580,21 @@ export namespace pages {
...
     export function isSupported(): boolean {
       return runtime.supports.pages ? (runtime.supports.pages.backStack ? true : false) : false;
     }
   }
 
+  /**
+   * @hidden
+   * Hide from docs
+   * ------
+   * Namespace to interact with the full-trust part of the SDK. Limited to 1P apps
```
Comment (erinha) [inline packages/teams-js/src/public/pages.ts:596]: @erikadoyle is this a sufficient summary of full trust? I'm not sure if we have something more informative to use honestly.

**Review by erinha — DISMISSED** [review]
One small clarification question for Erika, but looks good overall!

File: packages/teams-js/src/public/pages.ts (hunk 5 above)
Comment (erikadoyle) [reply packages/teams-js/src/public/pages.ts:596]: Good enough for now I think, esp. since its internal.

**Comment by erikadoyle** [comment]
THanks @jadahiya-MSFT! LGTM

**Review by TrevorJoelHarris — APPROVED** [review]

**Review by shrshindeMSFT — APPROVED** [review]

**Pull request merged**